### PR TITLE
Add release notes ocis 6.0.0 (rolling)

### PR DESCRIPTION
**Pull request by tbsbdr**
## ocis 6.0.0 rolling release notes
The focus of this release is about its self (the rolling release). Because of that I added some more infos that are usually not put into release notes, but since the rolling release needs to be introduced, I added a paragraph what rolling is and how to get it incl. a link to the online demo to get hooked.

## Contents
- General
- Rolling Release
- Online Demo
- How to Get the Rolling Release
  - Docker
  - Binary
  - Source Code on GitHub
- Web Office: Collabora Secure View
- Markdown Editor: ToastUI
- View e-Books: ePub Reader
- Diagramming Tool draw.io
- Show Audio and Image EXIF Metadata
- Other Notable Changes
- Migration
- Breaking Changes
- Deprecation

### CODE REVIEW

**Review by AlexAndBear — APPROVED** [review]
🤩

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:22]:
```suggestion
IMPORTANT: This is a Rolling Release. Please check the https://owncloud.dev/ocis/release_roadmap/#release-types[documentation] to see if this release type is right for your use case. There is no upgrade path from previous releases, so this version is therefore only suitable for you if you start with a clean slate.
```

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:36]: doesn't run on rolling

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:77]:
```suggestion
We have integrated the ToastUI markdown editor, providing users with robust capabilities to edit on markdown (.md or .markdown) files. Markdown files offer distinct advantages such as simplicity in formatting text using plain text syntax. The big advantage of markdown lies in its platform-independence as you don't need a complex or commercial App to edit these files. They are just plain simple and ultra useful. The ToastUI markdown editor enhances this experience with features like inline preview, syntax highlighting and an editor toolbar to help you with the markdown syntax. Collaborative (live) editing is currently not available. 
```

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:81]:
```suggestion
Infinite Scale now includes per default the ability to open ePub e-books via the integrated ePub reader. This feature supports the EPUB 3.0 format as specified by the International Digital Publishing Forum (IDPF). It utilizes the Epub.js library, a versatile JavaScript tool for rendering ePub documents in the browser across various devices.
```

If you want to mention someone, it was built by @AlexAndBear - same as the ToastUI integration. But honestly I wouldn't mention members of the core team. ;-)

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:91]:
```suggestion
Introducing new metadata details, Infinite Scale now features an `EXIF`` panel that displays image metadata whenever available. These metadata are shown from image `EXIF` data:
```

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:122]: I'd love to see this among the detailed items above. Including a link to the [awesome-ocis](http://github.com/owncloud/awesome-ocis) repo (which then also contains a link to the docs) and some words about our extension system. It is in quite good shape meanwhile. Would be worth writing something about this (I can do or at least help, but not today)

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:128]: @JammingBen has also made more really noticeable improvements regarding the web ui initial loading time / performance. He had some numbers for the lighthouse score improvement for the web ui, I think it was something like before: 50, after: 94 (out of 100).

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:140]: Do before merging this? Or followup? Also, this is `@micbar` on github 😁  Unless you want to involve our dear friend @mbarz :heart:

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:145]: Here as well: do before merging this? Or followup?

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:148]: We don't have release notes for ocis `5.0.5`? 🫨

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:117]: more things that come to my mind:
- a ton of SSE events, making the UI very much reactive to what happens server side... feels awesome already.
- we added more languages to the language switcher on the account page and a link to the transifex project if people want to help translate.
- state of the left and right sidebar is persisted now. If you open the right sidebar it will stay open until you close it. Same for the collapsed state of the left sidebar. This makes the fact that the right sidebar has been made available globally (= e.g. in editor apps and with the toggle button in the topbar) even more handy.

With all three things mentioned above @AlexAndBear can help you write something, because he was either heavily involved or even built it mostly on his own (SSE ❤️ )

**Review by kulmann — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (phil-davis) [inline modules/ROOT/pages/ocis_release_notes.adoc:121]:
```suggestion
* Enhancement - Change Cors default settings. We have changed the default CORS settings to set Access-Control-Allow-Origin to the OCIS_URL if not explicitly set and Access-Control-Allow-Credentials to false if not explicitly set. #8514 #8518
```

File: modules/ROOT/pages/ocis_release_notes.adoc
Comment (phil-davis) [inline modules/ROOT/pages/ocis_release_notes.adoc:124]:
```suggestion
* Enhancement - Allow public shares to be resolved without the ocs tokeninfo endpoint. Instead of querying the /v1.php/apps/files_sharing/api/v1/tokeninfo/ endpoint, a client can now resolve public and internal links by sending a PROPFIND request to /dav/public-files/{sharetoken} Authenticated clients accessing an internal link are redirected to the "real" resource (`/dav/spaces/{target-resource-id} Authenticated clients are able to resolve public links like before. For password protected links they need to supply the password even if they have access to the underlying resource by other means. Unauthenticated clients accessing an internal link get a 401 returned with WWW-Authenticate set to Bearer (so that the client knows that it needs to get a token via the IDP login page. Unauthenticated clients accessing a password protected link get a 401 returned with an error message to indicate the requirement for providing the link's password. #8858 #8926 cs3org/reva#4653
```

**Pull request closed without merging**
